doc/rst/technotes/globalvars.rst

```diff
@@ -0,0 +1,47 @@
+.. _readme-globalvars:
+
+==================================
+Global Variables
+==================================
+
+.. warning:: The variables defined here are not part of the Chapel Standard and 
+             they can be changed, renamed, or removed at any point in the future.
+
+The Chapel Compiler defines some variables that are available to all Chapel programs. 
+These variables are used by the compiler itself and the standard Chapel libraries/packages 
+to provide some features/optimizations based on compilation flags. 
+All of these variables are compile-time constants.
+
+Variables
+---------
+
+**boundsChecking**
+
+    This variable is *true* when ``--bounds-checks`` is enabled. 
+    It is used by libraries to enable/disable runtime bounds checks while 
+    accessing ranges, arrays, or domains.
+
+**castChecking**
+
+    This variable reflects whether ``--cast-checks`` flag is enabled.
```

Comment:
I think it should be "the ``--cast-checks`` flag"

Reply:
Sure Maam I will update it.